apps/user-profile/src/components/common/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  DropdownMenu,
+  DropdownMenuTrigger,
+  DropdownMenuContent,
+  DropdownMenuItem,
+} from '@devlaunchers/components/components/DropdownMenu';
+function DropDownMenu({ menuItems = [], country, onCountryChange }) {
+  return (
+    <DropdownMenu>
+      <p className="text-xs leading-4 block w-full box-border font-normal pd-3">
+        Country
+      </p>
+      <DropdownMenuTrigger asChild size="100%">
+        <div className="group">{country || 'Select...'}</div>
+      </DropdownMenuTrigger>
+      <DropdownMenuContent size="large">
+        {menuItems.map((item) => (
+          <DropdownMenuItem
+            key={item.code}
+            onSelect={() => onCountryChange(item.name)}
```

Comment:
Similar thing here, this is coupled to the country solution. What if someone wants to use this component in other places where `code` is not available.